functions/reporting_functions.sql

```diff
@@ -77,4 +77,81 @@ END
 $$ LANGUAGE plpgsql;
 
 
+
+CREATE OR REPLACE FUNCTION staffing_and_billing_overview(in_from_date date, in_to_date date)
+RETURNS TABLE (
+  year INT,
+  week INT,
+  from_date date,
+  to_date date,
+  available_hours NUMERIC,
+  billable_hours NUMERIC,
+  planned_fg NUMERIC,
+  actual_available_hours NUMERIC,
+  actual_billable_hours NUMERIC,
+  actual_fg NUMERIC,
+  deviation_available_hours NUMERIC,
+  deviation_billable_hours NUMERIC,
+  deviation_fg NUMERIC,
+  visibility NUMERIC
+) AS
+$$
+BEGIN
+  RETURN QUERY (
+
+select
```

Comment:
While I personally prefer lowercase, the style we have used so far, and also in the rest of your code, is uppercase keywords.